src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
+
+const ERROR_MESSAGE = 'Invalid values for this figure';
```

Comment:
The better idea is to have more detailed error message (imagine user trying to guess what went wrong)